test/e2e/framework/util.go

```diff
@@ -3455,41 +3473,38 @@ func isNodeConditionSetAsExpected(node *v1.Node, conditionType v1.NodeConditionT
 				if wantTrue {
 					if (cond.Status == v1.ConditionTrue) && !hasNodeControllerTaints {
 						return true
-					} else {
-						msg := ""
-						if !hasNodeControllerTaints {
-							msg = fmt.Sprintf("Condition %s of node %s is %v instead of %t. Reason: %v, message: %v",
-								conditionType, node.Name, cond.Status == v1.ConditionTrue, wantTrue, cond.Reason, cond.Message)
-						} else {
-							msg = fmt.Sprintf("Condition %s of node %s is %v, but Node is tainted by NodeController with %v. Failure",
-								conditionType, node.Name, cond.Status == v1.ConditionTrue, taints)
-						}
-						if !silent {
-							Logf(msg)
-						}
-						return false
 					}
-				} else {
-					// TODO: check if the Node is tainted once we enable NC notReady/unreachable taints by default
-					if cond.Status != v1.ConditionTrue {
-						return true
+					msg := ""
```

Comment:
So your PR to this point was pure linting, but some of this looks like style choice, and I'd argue the large else{} block is easier to read.  I get why you eliminated due to the returns, but when reading a mountain of code fast else{}'s help.

Reply:
I can see @timothysc's point, but that could depend on developers' background and thinking.
That means sometimes it is difficult to make concrete consensus and that is reason why we have golint check to make it simple and easy.
Ideally it is great to have some skipping flag for golang like python's `#noqa` for enabling linting check with partial skip.

Reply:
Thank you for reviewing.
I returned this part to original.